examples/image_streamer/golden_image.pp

```diff
@@ -0,0 +1,79 @@
+################################################################################
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# NOTE: As with all resources, the found ensurable accepts a data as an optional filter field.
+
+image_streamer_golden_image{'golden_image_1':
+  ensure => 'present',
+  data   => {
+    name         => 'Golden_Image_1',
+    description  => 'Golden Image created from the deployed OS Volume',
+    imageCapture => true,
+    osVolumeURI  => 'OSVolume-7',
+    buildPlanUri => 'Build Plan Name'
+  }
+}
+
+image_streamer_golden_image{'golden_image_2':
+  ensure => 'present',
+  data   => {
+    name              => 'Golden_Image_2',
+    description       => 'Golden image added from the file that is uploaded from a local drive',
+    golden_image_path => 'golden_image.zip'
```

Comment:
If it does accept an absolute path, I'd like to have that as a comment on the example. If it does not.. probably should.